Calculator/Calculator/Model/CalculatorItemQueue.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  CalculatorItemQueue.swift
+//  Calculator
+//
+//  Created by Minseong Kang on 2023/05/30.
+//
+
+extension Int: CalculateItem { }
+
+protocol CalculateItem { }
+
+struct CalculatorItemQueue<Element: CalculateItem> where Element: Equatable {
+	private var elements: [Element] = []
+	
+	func getElementsForTest() -> [Element] {
+		return elements
+	}
+	
+	mutating func enqueue(_ value: Element) {
+		elements.append(value)
+	}
+	
+	mutating func dequeue() -> Element? {
+		if elements != [] {
+			return elements.removeFirst()
+		} else {
+			return nil
+		}
```

Comment:
사바사겠지만, 저였다면 요런식으로 표현하는게 좀 더 간단해 보이네요! 어떻게 생각하세요?
```suggestion
		elements.isEmpty ? nil : elements.removeFirst()
```

Reply:
원비께서 피드백주신 삼항 연산자가 훨씬 깔끔하고 가독성이 좋습니다. 가독성이 좋은 코드를 만들기 위해 노력하겠습니다. 🙋‍♂️